node-src/lib/getPackageManager.ts

```diff
@@ -13,7 +13,7 @@ export const getPackageManagerRunCommand = async (args: string[]) => {
 
 // e.g. `8.19.2`
 export const getPackageManagerVersion = async (packageManager: string) => {
-  const { stdout } = await execa(packageManager || (await getPackageManagerName()), ['--version']);
+  const { stdout } = await execa(packageManager, ['--version']);
```

Comment:
What does the caller look like on this?  Can we be sure `packageManager` will actually be a non-empty `string`?

Reply:
The caller grabs the package manager before calling `getPackageManagerVersion` [here](https://github.com/chromaui/chromatic-cli/blob/c7a3f96ae59431e3edde1b757b0cb5cf0d45ee0d/node-src/tasks/initialize.ts#L55-L59) which feels like the right flow rather than magically grabbing the package manager if it's not passed in.

I think you make a good point though, we should handle `packageManager: ''` directly instead of letting `execa` throw for us. I'll add that layer quick!